ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -390,6 +390,121 @@ const PresetSelector = ({hasMicroarrayData, setSelectedDiskSize, setSelectedMach
   </PopupTrigger>;
 };
 
+const StartStopRuntimeButton = ({workspaceNamespace}) => {
+  const [status, setRuntimeStatus] = useRuntimeStatus(workspaceNamespace);
+
+  const rotateStyle = {animation: 'rotation 2s infinite linear'};
+  const {altText, iconShape = null, styleOverrides = {}, onClick = null, } = switchCase(status,
+    [
+      RuntimeStatus.Creating,
+      () => ({
+        altText: 'Runtime creation in progress',
```

Comment:
I notice in the video that the alt text is bouncing with the rotation animation as the effective width/position of the icon changes. Would be good to fix this